hknweb/events/templates/events/show_details.html

```diff
@@ -64,21 +64,31 @@ <h1 id="event-detail-title"> {{ event.name }} </h1>
       <p class="rsvp-list">You have not rsvp'd</p>
     {% endif %}
 
-    
-    <h3 class="rsvp-list-title">RSVPs</h3>
-    
+
+    {% if limit %}
+      <h3 class="rsvp-list-title">RSVPs ({{ rsvps.count }} / {{ limit }})</h3>
+    {% else %}
+      <h3 class="rsvp-list-title">RSVPs</h3>
+    {% endif %}
+
     {% if rsvps %}
-      {% if limit %}
-        <p class="rsvp-list"> Count: {{ rsvps.count }} / {{ limit }} </p>
-      {% else %}
-        <p class="rsvp-list"> Count: {{ rsvps.count }} </p>
-      {% endif %}
       {% for rsvp in rsvps %}
-        <p class="rsvp-list">{{ rsvp.user.first_name }} {{ rsvp.user.last_name }} ({{ rsvp.user }})</p>
+        <p class="rsvp-list">{{ rsvp.user.first_name }} {{ rsvp.user.last_name }} ({{ rsvp.user.email }})</p>
       {% endfor %}
-    {% elif not rsvpd %}
+    {% else %}
       <p class="rsvp-list">No rsvps.</p>
     {% endif %}
+
+    {% if waitlist %}
+      {% if waitlisted %}
+        <h3 class="rsvp-list-title">Waitlist (your position: {{ waitlist_position }})</h3>
```

Comment:
The fact that you are waitlisted should be more obvious. We should not display "You are rsvp'd [checkmark]" since that is misleading, the person might think they are done. It should display something like "You are waitlisted [... image]" Making the text a special color won't hurt either.